extensions/infinispan-embedded/runtime/src/main/java/io/quarkus/infinispan/embedded/runtime/graal/JChannelSubstitutions.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.infinispan.embedded.runtime.graal;
+
+import org.jgroups.Address;
+
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+
+/**
+ * These substitutions need to be revisited on each new Quarkus/GraalVM release, as some methods may become
+ * supported. The better option currently is to use -H:+ReportUnsupportedElementsAtRuntime, which moves potential issue
```

Comment:
I wouldn't suggest that as as "better option" YMMV

Reply:
This is just copied from Bela's stuff :) I can remove the comment.

Reply:
yes please remove :)